suites/reef/rbd/tier-2_rbd_cache_scenarios.yaml

```diff
@@ -84,28 +84,18 @@ tests:
       polarion-id: CEPH-83573758
 
   - test:
-      abort-on-fail: true
+      desc: Install rbd-nbd and remove any epel packages
+      module: exec.py
+      name: Install rbd-nbd
```

Comment:
why is this test needed?

Reply:
Because baremetal setup does not install rbd-nbd package. I have already mentioned this in the description for this PR.